src/js/components/RangeInput/__tests__/RangeInput-test.js

```diff
@@ -105,4 +105,13 @@ describe('RangeInput', () => {
     expect(container.firstChild).toMatchSnapshot();
     expect(onChange).toBeCalledTimes(1);
   });
+
+  test('inputValue', () => {
+    const { container } = render(
+      <Grommet>
+        <RangeInput min={0} max={10} step={1} value={5} inputValue />
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
```

Comment:
What is this test actually testing? that the component was created and rendered? Can we also check that it accepts the user's input?

Reply:
Yes, I need to improve this test.